resources/startup.sh

```diff
@@ -38,58 +36,72 @@ if [[ ! -d ${OPENLDAP_CONFIG_DIR}/cn=config ]]; then
   echo "get domain and root password"
   # get domain and root password
   LDAP_ROOTPASS=$(doguctl random)
-  doguctl config -e rootpwd ${LDAP_ROOTPASS}
-  LDAP_ROOTPASS_ENC=$(slappasswd -s $LDAP_ROOTPASS)
+  doguctl config -e rootpwd "${LDAP_ROOTPASS}"
+  LDAP_ROOTPASS_ENC=$(slappasswd -s "$LDAP_ROOTPASS")
+  export LDAP_ROOTPASS_ENC
+
   LDAP_BASE_DOMAIN=$(doguctl config --global domain)
   LDAP_DOMAIN=$(doguctl config --global domain)
+  export LDAP_BASE_DOMAIN
```

Comment:
Why is it necessary to export this variable? Is it even used anywhere? And if it is used, can it maybe be replaced by LDAP_DOMAIN, as they both hold the same value?